easybuild/easyconfigs/p/PANDAseq/PANDAseq-2.11-intel-2018a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'PANDAseq'
+version = '2.11'
+
+homepage = 'https://github.com/neufeld/pandaseq'
+description = """PANDASEQ is a program to align Illumina reads, optionally with PCR primers embedded in the sequence, and reconstruct an overlapping sequence."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+# download from https://github.com/neufeld/pandaseq/archive/v2.11.tar.gz
+source_urls = ['https://github.com/neufeld/%(namelower)s/archive/v%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+
+checksums = [('sha512', '1c37b8f777b698fe02df0248b9352c9550af7da0c739d0186f9973d4b4fcc30febc9c6345b098bd96ffb5dce703ccd201850fc543caad3e9da09b0e38fec4396')]
+
+
+dependencies = [
+    ('libtool', '2.4.6'),
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.11'),
+]
+
+builddependencies = [('Autoconf', '2.69','',('GCCcore','6.4.0'))]
```

Comment:
@ajaust Please use the `Autotools` bundle instead, and no need to hardcode to `GCCcore`:

```python
builddependencies = [('Autotools', '20170619')]

Reply:
I have to check what is going on. The hardcoded `GCCcore` can be removed (sorry about that). When using the `Autotools` module instead of `Autoconf`+`libtools` the generated `configure` script is different and doesn't work anymore. I'm not sure what is going on there.

Reply:
You can fix that issue by also including `pkg-config` as a build dependency, cfr. https://github.com/neufeld/pandaseq/issues/50

```python
builddependencies = [
    ('Autotools', '20170619'),
    ('pkg-config', '0.29.2'),
]
```

I just tested myself: same problem as reported in that issue occurs without `pkg-config`, works fine with `pkg-config`

Reply:
You are right. Thanks for pointing me to that. I have updated the easyconfig file accordingly.